spec/card_spec.rb

```diff
@@ -1,27 +1,17 @@
 require './lib/card'
 
 RSpec.describe Card do
-  it 'exists' do
-    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
-
-    expect(card).to be_instance_of(Card)
-  end
-
-  it 'has a question' do
-    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
-
-    expect(card.question).to eq("What is the capital of Alaska?")
+  before(:each) do
+    @card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
   end
 
-  it 'has an answer' do
-    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
-
-    expect(card.answer).to eq("Juneau")
+  it 'exists' do
+    expect(@card).to be_instance_of(Card)
   end
 
-  it 'has a category' do
-    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
-
-    expect(card.category).to eq(:Geography)
+  it 'has a question' do
```

Comment:
Your test is described as checking for `has a question`, but your `expect` statements actually do 3 things: 
1. Checks for question 
2. Checks for its answer
3. Checks for its category. 

It's fine to put multiple `expect` statements in one test, but it should always be if the test is describing all 3 things, or makes that test more robust in some way. For instance, you could rename this test to 
`it "has a question, answer, and category" do`
or
`it "has attributes" do`